content/applications/finance/accounting/reporting/overview/data_inalterability.rst

```diff
@@ -0,0 +1,59 @@
+================================
+Data inalterability check report
+================================
+
+Tax authorities in some countries require companies to prove their posted accounting entries are
+inalterable, meaning that once an entry has been posted, it can no longer be changed.
+
+To do so, Odoo can use the SHA-256 algorithm to create a unique fingerprint for each posted entry.
+This fingerprint is called a hash. The hash is generated by taking an entry's essential data (the
+values of the `date`, `journal_id`, `company_id`, `debit`, `credit`, `account_id`, and `partner_id`
+fields), concatenating it, inputting it to the SHA-256 hash function, which then outputs a fixed
+size (256-bit) string of characters. The hash function is deterministic, meaning that the same input
+always creates the same output: any minor modification to the original data will completely change
+the resulting hash. Consequently, the SHA-256 algorithm is often used, among others, for data
+integrity verification purposes.
```

Comment:
```suggestion
Tax authorities in some countries require companies to prove their posted accounting entries are
inalterable, meaning that once an entry has been posted, it can no longer be changed.

To do so, Odoo can use the SHA-256 algorithm to create a unique fingerprint for each posted entry.
This fingerprint is called a hash. The hash is generated by taking an entry's essential data (the
values of the `date`, `journal_id`, `company_id`, `debit`, `credit`, `account_id`, and `partner_id`
fields), concatenating it, and inputting it to the SHA-256 hash function, which then outputs a fixed
size (256-bit) string of characters. The hash function is deterministic, meaning that the same input
always creates the same output: any minor modification to the original data will completely change
the resulting hash. Consequently, the SHA-256 algorithm is often used, among others, for data
integrity verification purposes.
```